test/e2e/specs/default-config/recordedit/foreign-key-dropdown.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { test, expect } from '@playwright/test';
+
+// locators
+import RecordeditLocators from '@isrd-isi-edu/chaise/test/e2e/locators/recordedit';
+
+// utils
+import { getCatalogID } from '@isrd-isi-edu/chaise/test/e2e/utils/catalog-utils';
+import { testSubmission } from '@isrd-isi-edu/chaise/test/e2e/utils/recordedit-utils';
+
+const testParams = {
+  table_name: 'fkey-dropdown',
+  schema_name: 'fk-display-mode',
+  table_displayname: 'fkey-dropdown',
+  fk1_name: 'Y8Fm0o1t3FcHt0S8UjXs6A',
+  fk2_name: 'jZz6GY0Bq-0EXpzflh6zZg',
+  fk3_name: '-wCXD7GyYPDYVhTgm9in3A',
+  column_names: ['fk1 dropdown', 'fk2 dropdown', 'fk3 dropdown'],
+  column_values: [['two', 'five', 'six']]
+};
+
+test.describe(`Create a record, For table ${testParams.table_name}`, () => {
```

Comment:
This `test.describe` is not needed and it's just making the report lengthier. You should just remove it. So then this file will just have the `test` that you have.

You could even rewrite the `test` description to something like `foreignkey dropdown support` or something like that.